Week1/js-exercises/ex2-aboutMe.js

```diff
@@ -1,11 +1,14 @@
-/**
- 
- ** Exercise 2: About me **
+const body = document.querySelector('body');
+body.style.fontFamily = "Arial, sans-serif";
+const ul = body.children[1].children;
```

Comment:
here you should've used a selector
```suggestion
const ul = document.querySelector('ul');
```

Reply:
Ok 👍🏻, I thought  using children and siblings for selector better than directly selectors like ```
Document.querySelector, OR getElementById.
```